clients/java/zts/core/src/main/java/com/yahoo/athenz/zts/ZTSClient.java

```diff
@@ -327,6 +334,23 @@ public static void lookupZTSUrl() {
             }
         }
     }
+
+    public static void initZtsFileUtil() {
+        ztsFileUtil = new ZTSFileUtil(getAccessTokenSignKeyResolver());
+        ztsFileUtil.preload();
+    }
+
+    public static void setAccessTokenSignKeyResolver(JwtsSigningKeyResolver jwtsSigningKeyResolver) {
```

Comment:
I'm not sure why we're only setting the resolver here and not doing anything else since it means after I do this I also need to call initZtsFileUtil to set the resolver and call preload. So why just within this call we reload the file util object ourselves instead of asking the user to do it.

Reply:
Having setAccessTokenSignKeyResolver and getAccessTokenSignKeyResolver is for setting mock resolver in unit test

Reply:
but we still need to make them usable since they're declared as public. so rather than just having a resolver kept as a member variable, we can call the init method again to reinitialize the file utility and you don't need the getAccessTokenSignKeyResolver() at all.

This also gives the client the flexibility for someone who doesn't want to use the conf file but define their own resolver, so that they can do so and pass it to the client.